include/client.h

```diff
@@ -67,12 +66,13 @@ struct Attributes {
 
     // Data value
     union {
-      std::string str_v;  // for both STRING and BYTES
       int64_t int64_v;
       double double_v;
       bool bool_v;
       std::chrono::time_point<std::chrono::system_clock> time_v;
     } value;
+    // Outside of union since it is not easy for union to support it.
+    std::string str_v;  // for both STRING and BYTES
```

Comment:
It did passed the test.  OK,  I will move it out to be safe since it does have a constructor.